src/main/java/nc/students/ayaz/model/Video.java

```diff
@@ -1,18 +1,27 @@
-package nc.students.ayaz;
+package nc.students.ayaz.model;
 
 import cobertura.IgnoreDuringCodeCoverage;
 
 import java.util.ArrayList;
 import java.util.List;
 
-class Video {
+public class Video {
```

Comment:
More info:
http://stackoverflow.com/questions/31094354/jersey-no-suitable-constructor-found-for-type-simple-type-class-thing-can-n